core/metadata.h

```diff
@@ -88,12 +41,7 @@ static inline int IsValidOffset(mt_offset_t offset) {
   return (offset >= 0);
 }
 
-
-enum mt_access_mode {
-  MT_READ = 0,
-  MT_WRITE,
-  MT_UPDATE
-};
+enum mt_access_mode { MT_READ = 0, MT_WRITE, MT_UPDATE };
```

Comment:
The type name should be changed..? (not sure)

With "enum class", we can remove the ugly MT_ prefix